doc/Reference/Reactive/state.md

```diff
@@ -164,3 +226,169 @@ This gives you the ability to update a _state_, including the metadata.
 
 > [!NOTE]
 > This is the raw way to update a state and is designed for advanced usage and should probably not be used directly in apps.
+
+### Usage of States
+
+States are advanced Feeds. As such, they can also be awaited directly:
+
+```csharp
+public static TaskAwaiter<T> GetAwaiter<T>(this IState<T> state);
+```
+
+For example:
+
+```csharp
+City currentCity = await this.CurrentCity;
+```
+
+### Binding the View to a State
+
+States are built to be cooperating with the data-binding engine. A State will automatically update its value when the user changes data in the View bound to this State.
+
+1. In an MVUX app (read [How to set up an MVUX project](xref:Uno.Extensions.Mvux.HowToMvuxProject)), add a Model class with a State as follows:
+
+```csharp
+    public partial record SliderModel
+    {
+        // create a state with an initial random double value between 0 and 1, multiplied by 100.
+        public IState<double> SliderValue => State.Value(this, () => Random.Shared.NextDouble() * 100);
+    }
+```
+
+1. Replace all child elements in the _MainPage.xaml_ with the following:
+
+```xml
+    <Page
+        x:Class="SliderApp.MainPage"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:SliderApp">
+        <Page.DataContext>
+            <local:SliderViewModel />
+        </Page.DataContext>
+
+        <StackPanel>
+            <StackPanel Orientation="Horizontal" Spacing="5">
+                <TextBlock Text="Current state value:" />
+                <TextBlock Text="{Binding SliderValue}" />
+            </StackPanel>
+
+            <Border Height="1" Background="DarkGray" />
+
+            <TextBlock Text="Set state value:"/>
+            <Slider Value="{Binding SliderValue, Mode=TwoWay}" />
+        </StackPanel>
+    </Page>
+```
+
+In this scenario, the `DataContext` is set to an instance of the `SliderViewModel` class, which is the generated ViewModel for the `SliderModel` record.
+
+1. When you run the app, moving the `Slider` instantly affects the upper `TextBox`. The `Silder.Value` property has a two-way binding with the `SliderValue` State, so any change to the Slider immediately updates the State value, which in turn affects the data-bound `TextBlock` on top:
+
+    ![A video of the previous slider app in action](/Learn/Mvux/Assets/SliderApp-1.gif)
```

Comment:
I think the gif path is incorrect:

![image](https://github.com/user-attachments/assets/e3085dcc-7d40-4836-8416-168c205bfebc)